modules/manage/pages/audit-logging.adoc

```diff
@@ -0,0 +1,46 @@
+= Audit Logging
+:description: Learn how to use Redpanda's audit logging capabilities.
+
+Many scenarios for streaming data include the need for fine-grained auditing of user activity related to the system. This is especially true for regulated industries such as finance, healthcare, and the public sector. Complying with PCI DSS v4 standards, for example, requires verbose and detailed activity auditing, alerting, and analysis capabilities.
+
+Redpanda's auditing capabilities support auditing both administrative and operational interactions with topics and with users. We comply with the open cybersecurity schema framework, providing a predictable and extensible solution which works seamlessly with industry standard tools.
```

Comment:
It may be helpful to provide a link to the OCSF schema page (https://schema.ocsf.io/) and/or github project https://github.com/ocsf.